src/main/pg/displayport_profiles.c

```diff
@@ -30,7 +30,14 @@
 
 #if defined(USE_MSP_DISPLAYPORT)
 
-PG_REGISTER(displayPortProfile_t, displayPortProfileMsp, PG_DISPLAY_PORT_MSP_CONFIG, 1);
+PG_REGISTER_WITH_RESET_FN(displayPortProfile_t, displayPortProfileMsp, PG_DISPLAY_PORT_MSP_CONFIG, 1);
```

Comment:
```suggestion
PG_REGISTER_WITH_RESET_FN(displayPortProfile_t, displayPortProfileMsp, PG_DISPLAY_PORT_MSP_CONFIG, 2);
```

Reply:
Is the version for the structure, which hasn’t changed, or the initialisation values? I thought it was the former.

Reply:
My feeling tells me it is alright if the structure didn't change at all but I wasn't sure so I marked it, just in case.